### PR TITLE
refactor(frontend): reduce code duplications using objects

### PR DESCRIPTION
**Pull request by tamirGer**

### CODE REVIEW

File: public/src/pages/main/Header/Nav.tsx [hunk 1]
```diff
@@ -93,8 +93,8 @@ export const Nav = () => {
   return (
     <nav className="nav-menu d-none d-lg-block">
       <ul>
-        {menu.map((item, i) =>
-          !item.admin || isAdminUser ? menuItem(item, i) : <></>
+        {menu.map(
+          (item, i) => (!item.admin || isAdminUser) && menuItem(item, i)
```
Comment (derevnjuk) [inline public/src/pages/main/Header/Nav.tsx:97]: Please revert the changes, the preferred way is to use the ternary operator

**Review by derevnjuk — CHANGES_REQUESTED** [review]

File: public/src/pages/main/Header/Nav.tsx (hunk 1 above)
Comment (tamirGer) [reply public/src/pages/main/Header/Nav.tsx]: @derevnjuk done

**Review by derevnjuk — APPROVED** [review]

**Comment by derevnjuk** [comment]
@tamirGer please resolve the conflicts before the merge

**Comment by tamirGer** [comment]
@derevnjuk Should be good to merge now :)
